nrm_django/grants/admin.py

```diff
@@ -141,6 +144,26 @@ class GrantAdmin(admin.ModelAdmin):
         ),
     )
 
+    def get_urls(self):
+        urls = super().get_urls()
+        my_urls = [
+            path('my_view/<username>', self.admin_site.admin_view(CustomView.as_view())),
+        ]
+        return my_urls + urls 
+
+
+class CustomView(ListView):
```

Comment:
Neil, what are you thinking for a mechanism to select a name from the 8000? We could do this with just a `list_filter` but the name selection would be limited to a list or dropdown. We could also make it a search, which would work well but not be that intuitive. Are you thinking there'a a future state where we auto-show your grants when you're authed?

None of this is particularly relevant to this review, i'm just wanting to be sure I understand

Reply:
This one is just the view at the url e.g. `/admin/grants/grant/my_view/ECHEN`. If you know the names from `created_by` you can see the list for each person.

I imagine the eventual UX is probably either a select box or grabbing the username of whoever is logged in.